services/nltk/posTagger.py

```diff
@@ -0,0 +1,23 @@
+import sys, os, nltk
+
+tmpBody = "Sentence; POS tags\n"
+
+inputFile = open("/root/input/text.txt", "r")
```

Comment:
When the data is downloaded you don't know the file name, it can be anything. A better solution would be to scan the directory and parse all the txt files in it